binder/environment.yml

```diff
@@ -17,6 +17,7 @@ dependencies:
   - nomkl
   - h5py
   - py-xgboost
+  - xarray
```

Comment:
We're working on this in https://github.com/conda-forge/xarray-feedstock/pull/37. For now, I've moved xarray to the pip section to avoid pulling unused option deps. Note, you already have scipy in the image so the add is just netcdf and a few other small IO libs.

Reply:
Okay, after playing with this for a bit, I don't think this is a big deal. The added weight is just about 8 MB. This will go down after the next xarray release.

```
The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    curl-7.62.0                |       h74213dd_0         135 KB  conda-forge
    libcurl-7.62.0             |       hbdb9355_0         535 KB  conda-forge
    hdf4-4.2.13                |       h951d187_2         970 KB  conda-forge
    libnetcdf-4.6.1            |      h9cd6fdc_11         1.3 MB  conda-forge
    netcdf4-1.4.2              |   py36hac939d9_0         2.0 MB  conda-forge
    krb5-1.16.2                |       hbb41f41_0         1.4 MB  conda-forge
    xarray-0.10.9              |           py36_0         775 KB  conda-forge
    bottleneck-1.2.1           |   py36h7eb728f_1         309 KB  conda-forge
    cftime-1.0.2.1             |   py36h7eb728f_0         1.1 MB  conda-forge
    libssh2-1.8.0              |       h5b517e9_2         240 KB  conda-forge
    ------------------------------------------------------------
                                           Total:         8.6 MB
```

Reply:
OK, fair enough.  Thanks for diving into this @jhamman

Reply:
this is even less of a problem now that xarray 0.11 (released yesterday) doesn't include all possible optional dependencies via conda-forge.